### PR TITLE
Rollup of VSD to Shared Package

### PR DESCRIPTION
**Pull request by Alixers505**
https://www.pivotaltracker.com/story/show/171697505

### CODE REVIEW

File: sites/vision-systems.com/server/templates/content/index.marko [hunk 1]
```diff
@@ -1,170 +1,10 @@
...
-    </marko-web-resolve-page>
-  </@foot>
-</marko-web-content-page-layout>
+<shared-content-page-default-layout
+  id=id
+  type=type
+  page-node=pageNode
+>
+  <@right-rail ad-layout="variant-a" />
+  <@load-more flow="card-deck" limit=10 />
```
Comment (zarathustra323) [inline sites/vision-systems.com/server/templates/content/index.marko]: Only the dental sites use the card deck. Remove this line.

File: sites/vision-systems.com/server/templates/index.marko [hunk 2]
```diff
@@ -1,257 +1,132 @@
...
-  </@below-page>
-  <@foot>
-    <marko-web-resolve-page|{ data: section }| node=pageNode>
-      $ const aliases = hierarchyAliases(section);
-      <marko-web-gam-fixed-ad-bottom ...GAM.getAdUnit({ name: "lb2", aliases }) refresh-interval=15 scroll-offset=100 />
-    </marko-web-resolve-page>
-  </@foot>
-</marko-web-website-section-page-layout>
+
+  <@load-more name="card-deck" max-pages=0>
```
Comment (zarathustra323) [inline sites/vision-systems.com/server/templates/index.marko]: Remove the `name` attribute so the default load more is used.

File: sites/vision-systems.com/server/templates/index.marko [hunk 3]
```diff
@@ -1,257 +1,132 @@
...
+    <@query-params limit=10 skip=5 />
```
Comment (zarathustra323) [inline sites/vision-systems.com/server/templates/index.marko]: `limit=14 skip=5`

File: sites/vision-systems.com/server/templates/website-section/index.marko [hunk 4]
```diff
@@ -1,79 +1,12 @@
...
-    </marko-web-resolve-page>
-  </@foot>
-</marko-web-website-section-page-layout>
+<shared-website-section-page-default-layout
+  id=id
+  alias=alias
+  name=name
+  page-node=pageNode
+>
+  <@load-more name="card-deck">
```
Comment (zarathustra323) [inline sites/vision-systems.com/server/templates/website-section/index.marko]: The entire `<@load-more>` block can be removed.

e.g.
```marko
$ const { id, alias, name, pageNode } = data;

<shared-website-section-page-default-layout
  id=id
  alias=alias
  name=name
  page-node=pageNode
/>
```

File: sites/vision-systems.com/server/templates/index.marko [hunk 5]
```diff
@@ -1,257 +1,132 @@
...
-  <@head>
-    <marko-web-gtm-website-section-context|{ context }| alias=alias>
-      <marko-web-gtm-push data=context />
-    </marko-web-gtm-website-section-context>
-    <marko-web-resolve-page|{ data: section }| node=pageNode>
-      $ const aliases = hierarchyAliases(section);
-      $ const adSlots = {
-          "gpt-ad-lb1":   GAM.getAdUnit({ name: "lb1", aliases }),
-          "gpt-ad-rail1": GAM.getAdUnit({ name: "rail1", aliases }),
-          "gpt-ad-rail2": GAM.getAdUnit({ name: "rail2", aliases }),
```
Comment (brandonbk) [inline sites/vision-systems.com/server/templates/index.marko]: Add `"gpt-ad-rail3": GAM.getAdUnit({ name: "rail1", aliases }),` for use next to the Leaders block.  The `gpt-ad-rail3` portion can only be included once on the page, but the `rail1` adunit can be reused (it's a 300x250).

File: sites/vision-systems.com/server/templates/index.marko [hunk 6]
```diff
@@ -1,257 +1,132 @@
...
+        <shared-magazine-latest-issue-block publication-id="5ca78dbc75a2541c0e0041a9" />
       </div>
     </div>
     <div class="row">
       <div class="col-lg-8 mb-block">
         <leaders-home-page />
       </div>
       <div class="col-lg-4 mb-block">
-        <marko-web-gam-display-ad id="gpt-ad-rail2" />
+        <shared-gam-display-ad id="gpt-ad-rail2" />
```
Comment (brandonbk) [inline sites/vision-systems.com/server/templates/index.marko]: This should reference `gpt-ad-rail3`

File: sites/vision-systems.com/server/templates/content/index.marko [hunk 7]
```diff
@@ -1,170 +1,10 @@
...
-    </marko-web-resolve-page>
-  </@foot>
-</marko-web-content-page-layout>
+<shared-content-page-default-layout
+  id=id
+  type=type
+  page-node=pageNode
+>
+  <@right-rail ad-layout="variant-a" />
+  <@load-more limit=10 />
```
Comment (zarathustra323) [inline sites/vision-systems.com/server/templates/content/index.marko]: Delete `<@load-more>` completely

File: sites/vision-systems.com/server/templates/index.marko [hunk 8]
```diff
@@ -1,257 +1,142 @@
...
-  </@below-page>
-  <@foot>
-    <marko-web-resolve-page|{ data: section }| node=pageNode>
-      $ const aliases = hierarchyAliases(section);
-      <marko-web-gam-fixed-ad-bottom ...GAM.getAdUnit({ name: "lb2", aliases }) refresh-interval=15 scroll-offset=100 />
-    </marko-web-resolve-page>
-  </@foot>
-</marko-web-website-section-page-layout>
+
+  <@load-more  max-pages=0>
```
Comment (zarathustra323) [inline sites/vision-systems.com/server/templates/index.marko]: Remove extra space here

**Pull request merged**